flight-tracker-client/src/contexts/userContext.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {createContext, useEffect, useState} from 'react'
+
+interface IState {
+    id: string;
+    loaded: boolean;
+    empty: boolean;
+}
+
+export const UserContext = createContext({
+    user: { id: "", loaded: false, empty: false },
+    updateUser: (user: IState) => {},
+    removeUser: () => {}
+})
+
+const UserContextProvider = (props: any) => {
+    const [user, setUser] = useState<IState>({
+        id: "",
+        loaded: false,
+        empty: false    
+    });
+
+    useEffect(() => {
+        const localData: string | null = localStorage.getItem('user');
```

Comment:
can you explain the reason for the use of localStorage to me?

Reply:
is Context not giving you the same functionality? Is this temporary and will be removed when we use Mongo?